src/test/java/org/openelisglobal/storage/controller/SampleStorageRestControllerDisposalTest.java

```diff
@@ -147,297 +96,123 @@ public void testDisposeSampleItem_Returns400ForMissingMethod() throws Exception
     }
 
     @Test
-    public void testDisposeSampleItem_ClearsStorageAssignment() throws Exception {
-        // Arrange
-        String sampleItemExternalId = createTestSampleItem();
-        int numericId = getSampleItemNumericId(sampleItemExternalId);
-
-        // Create a storage assignment
-        jdbcTemplate.update(
-                "INSERT INTO sample_storage_assignment (id, sample_item_id, location_id, location_type, assigned_by_user_id, assigned_date, last_updated) VALUES (?, ?, ?, ?, ?, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP)",
-                10000, numericId, 10, "device", 1);
-
+    public void disposeSampleItem_ShouldClearStorageAssignment_WhenPresent() throws Exception {
+        String sampleItemExternalId = "EXT-1002";
         String requestBody = "{\"sampleItemId\":\"" + sampleItemExternalId
                 + "\",\"reason\":\"expired\",\"method\":\"autoclave\"}";
 
-        // Act
         this.mockMvc.perform(
                 post("/rest/storage/sample-items/dispose").contentType(MediaType.APPLICATION_JSON).content(requestBody))
                 .andExpect(status().isOk());
 
-        // Assert - assignment exists with NULL location
-        // (specs/001-sample-storage/spec.md FR-056)
-        Integer assignmentCount = jdbcTemplate.queryForObject(
```

Comment:
Thanks @Agaba-derrick ,
I think just not to lose all this different tes case scenearios , we can allow using jdbcTemplate to quesry for that specific info and keep the tests. 
Am just carefull not to alter the Tests Logic and to keep as much testes case scenarios as possible